python/lib/core/README.md

```diff
@@ -2,3 +2,169 @@
 Python package for core DMOD types, both concrete and abstract, that are depended upon by other DMOD Python packages and themselves have no dependencies outside of Python and its standard library.  
 
 Classes belong here if placing them in a more specialized package would cause undesired consequences, such as circular dependencies or transitive dependency on otherwise unnecessary packages.
+
+## `common`
+
+<b style="color: red">TODO:</b> Write information about the `dmod.core.common` package
```

Comment:
Github apparently doesn't render the `color` css attribute, which is a shame. At least it renders and is bold. I tried some other approaches, such as with tex and gh markdown, but they don't naturally render in an IDE. Having the statement in HTML at least renders, which is good enough until the content is provided.

As for whether to put the TODOs in docs, it:

a) demonstrates an intention to fill in documentation
b) serves as a blatant reminder that documentation should be written
c) is not the only location where we have TODOs in markdown

I'm going to go ahead and add in an umbrella ticket for this.

If we don't want TODOs in the docs, we should fill them in, not hide them elsewhere. The core README has 4 TODOs, but they are commented within the markdown themself some aren't rendered. As a result, it's not apparent that more work needs to be done unless the markdown "code" itself is scrutinized.

Reply:
> As a result, it's not apparent that more work needs to be done unless the markdown "code" itself is scrutinized.

What it really boils down to for me is, is the above effect a bug or a feature?  I think there will at least be a time in the future when we will want to do this:  something more akin to "put things away" than "hide things."  But we don't have to be that strict about it now if it helps us remember things more effectively.

Reply:
I'd call the required scrutinization (new word - patent pending, patent pending, patent pending) more of a bug, but not something to standardize for or against. We have close to about 200 tickets at the moment and I'm afraid that, with this readme being as buried as it is, that's it's highly unlikely that it's going to see any real attention. If _anyone_ is zooming around and sees a bunch of TODOs instead of the usual mostly blank page, that might serve as a reminder and get the ball rolling again after its been abandoned. With these stubs there, it should help guide people and allow slightly easier collaboration (just a big fat "Feel free to enter the blank"). It's the approach I prefer _here_, but not one that I think trumps any other or is any kind of grand idea.